evap/student/views.py

```diff
@@ -253,6 +253,8 @@ def render_vote_page(
     for_rendering_in_modal: bool = False,
 ) -> HttpResponse:
     language = request.GET.get("language", evaluation.main_language)
+    if language == "x":
+        language = "en"
```

Comment:
two thoughts:
1. we do not expect that `request.GET["language"]` would ever be "x", we only expect it for `evaluation.main_language`; can you change it so that only this case is checked?
2. please don't use "x" directly, instead use the `Evaluation.UNDECIDED_MAIN_LANGUAGE` constant